src/steamship/app/request.py

```diff
@@ -39,13 +39,17 @@ class Invocation(BaseModel):
 
 
 class Request(BaseModel):
-    """An request of a method on an app instance.
+    """A request as the Steamship Hosting Framework receives it from the Engine.
 
-    This is the payload sent from the public-facing App Proxy to the
-    private-facing app microservice.
+    This class is different from the other `Request` class:
+     * `steamship.base.request` represents a request from the Steamship Client
+     * this class represents a request from the Steamship Engine to a Steamship-hosted App/Plugin
+
+    It contains both an app/plugin invocation and also the client configuration in which that invocation
+    is intended to execute.
```

Comment:
This comment is definitely very helpful, but might we just want to also rename the class?

Reply:
Yeah -- either that or split off the app sdk into its own thing. Will punt for now figuring we'll circle back to this